examples/odop_solar.py

```diff
@@ -12,42 +14,57 @@ def __init__(self, n, dt):
 
     @staticmethod
     @ti.func
-    def random_around(center, radius):
-        # random number in [center - radius, center + radius)
-        return center + radius * (ti.random() - 0.5) * 2
+    def random_vector_in(rmax):
+        # create a random vector
+        a = ti.random() * math.tau
+        r = (ti.random()**0.7) * rmax  # For adjusting PDF
```

Comment:
```suggestion
        r = ti.random() * rmax
```

We'd better keep everything as simple as possible in this example.